PortalApplication/ApplicationRunner.swift

```diff
@@ -100,26 +100,20 @@ internal extension ApplicationRunner {
                             self.navigationState = nextNavigationState
                             
                             self.render(view: view)
+                            
+                            switch (thenSend) {
```

Comment:
|> operator is defined in PortalView, not PortalApplication, should i define it?